service/RTC/MediaType.js

```diff
@@ -1,15 +1,9 @@
 /**
- * Enumeration of RTC media stream types
- * @type {{AUDIO: string, VIDEO: string}}
+ * The audio type.
  */
-var MediaType = {
-    /**
-     * The audio type.
-     */
-    AUDIO: "audio",
-    /**
-     * The video type.
-     */
-    VIDEO: "video"
-};
-module.exports = MediaType;
\ No newline at end of file
+export const AUDIO = "audio";
```

Comment:
Hmm I know what you mean, but I noticed that we use this convention in few other places which have been converted to the ES6 already, so I tried to comply:
https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConferenceErrors.js
https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConferenceEvents.js